### PR TITLE
VR-5587 Enable Dataset tags CRUD

### PR DESCRIPTION
**Pull request by nhatsmrt**
Requires _update from https://github.com/VertaAI/modeldb/pull/1197/

### CODE REVIEW

File: client/verta/verta/_dataset_versioning/dataset.py [hunk 1]
```diff
@@ -113,22 +114,66 @@ def get_description(self):
...
+            All tags.
+
+        """
+        self._refresh_cache()
+        return self._msg.tags
 
     def del_tag(self, tag):
-        raise NotImplementedError
+        """
+        Deletes a tag from this dataset
```
Comment (convoliution) [inline client/verta/verta/_dataset_versioning/dataset.py]:
```suggestion
        Deletes a tag from this Dataset.
```

**Review by convoliution — CHANGES_REQUESTED** [review]
Small docstring request

File: client/verta/verta/_dataset_versioning/dataset.py (hunk 1 above)
Comment (nhatsmrt) [reply client/verta/verta/_dataset_versioning/dataset.py]: Fixed.

**Comment by nhatsmrt** [comment]
Conflict resolved.

**Review by convoliution — APPROVED** [review]

**Pull request merged**